mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/Strategy.scala

```diff
@@ -61,6 +61,8 @@ import org.apache.spark.mllib.tree.impurity.{Entropy, Gini, Impurity, Variance}
  * @param subsamplingRate Fraction of the training data used for learning decision tree.
  * @param useNodeIdCache If this is true, instead of passing trees to executors, the algorithm will
  *                       maintain a separate RDD of node Id cache for each row.
+ * @param canMergeChildren Merge pairs of leaf nodes of the same parent which
```

Comment:
A new parameter is added in Strategy class, which fails Mima tests. How to deal with it?

```bash
[error]  * synthetic method <init>$default$13()Int in object org.apache.spark.mllib.tree.configuration.Strategy has a different result type in current version, where it is Boolean rather than Int
```
[see failed logs](https://amplab.cs.berkeley.edu/jenkins/job/NewSparkPullRequestBuilder/3675/consoleFull)